frontend/apps/reader/modules/readerlink.lua

```diff
@@ -26,7 +24,30 @@ local ReaderLink = InputContainer:new{
 
 function ReaderLink:init()
     if Device:isTouchDevice() then
-        self:initGesListener()
+        self.ui:registerTouchZones({
+            {
+                id = "tap_link",
+                ges = "tap",
+                screen_zone = {
+                    ratio_x = 0, ratio_y = 0,
+                    ratio_w = 1, ratio_h = 1,
+                },
+                overrides = {
+                    "tap_forward",
+                    "tap_backward",
+                },
+                handler = function(ges) return self:onTap(_, ges) end,
+            },
+            {
+                id = "swipe_link",
+                ges = "swipe",
+                screen_zone = {
+                    ratio_x = 0, ratio_y = 0,
+                    ratio_w = 1, ratio_h = 1,
+                },
+                handler = function(ges) return self:onSwipe(_, ges) end,
```

Comment:
That new one might also need some override (paging_swipe, rolling_swipe...) I guess.